.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   create-tar:
     docker:
-       - image: quay.io/quintype/black-knight:bk-base
+       - image: quay.io/quintype/black-knight:bk-rubyupgrade
```

Comment:
`bk-rubyupgrade` -> `bk-ruby-2.6` ?

Reply:
we changed it to bk-rubyupgrade(new image with ruby version 2.6.0) because we wanted to keep a backup of the working image (bk-base) .